### PR TITLE
Add yast2-security regression test

### PR DESCRIPTION
**Pull request by paolostivanin**
- Related ticket: https://progress.opensuse.org/issues/49331
- Needles:
  - oS: https://github.com/os-autoinst/os-autoinst-needles-opensuse/pull/553
  - SLE: https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/1130
- Verification runs:
  - oS TW: http://d502.qam.suse.de/tests/1884 :heavy_check_mark: 
  - oS Leap: http://d502.qam.suse.de/tests/1885  :heavy_check_mark: 
  - SLE 15: http://d502.qam.suse.de/tests/1883  :heavy_check_mark: 
  - SLE 12.4: http://d502.qam.suse.de/tests/1880  :heavy_check_mark: 
  - SLE 12.3: http://d502.qam.suse.de/tests/1882  :heavy_check_mark: 
  - SLE 12.2: http://d502.qam.suse.de/tests/1881  :heavy_check_mark: 

The test will reside on the newly created `mau-extratests-yast2ui` (for `osd`)

### CODE REVIEW

**Comment by wnereiz** [comment]
The code is LGTM. But is it possible to check whether these configurations work as expected?

Eg. for `Minimum Acceptable Password Length` option, you set '8' here.  Would you consider to verify it by setting the password with length of 7 and 8 respectively, and check the different result?

**Comment by paolostivanin** [comment]
@wnereiz good point :) let me add some checks!

File: tests/yast2_gui/yast2_security.pm [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+use warnings;
+use testapi;
+
+sub run {
+    my $self = shift;
+    select_console "x11";
+
+    # Password Settings
+    $self->launch_yast2_module_x11("security", match_timeout => 120);
+    assert_and_click "yast2_security-pwd-settings";
```
[the remark this answers is not in the document]
Comment (paolostivanin) [reply tests/yast2_gui/yast2_security.pm]: Nice, I didn't know that :+1: 
But that would be a different thing though. Here we are testing "desktop" ui and not the terminal one :)

**Comment by paolostivanin** [comment]
I just found out that changing the password using `passwd` doesn't show any warning on SLE. The warning is only shown when changing the password using the YaST2 Users module...

**Comment by paolostivanin** [comment]
Agreed! I'll write that down on the poo :+1:

**Comment by paolostivanin** [comment]
:warning: to be merged after https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/7403

**Comment by dzedro** [comment]
Update `lib/main_common.pm`

**Comment by paolostivanin** [comment]
Verification runs after latest rebase:
SLE 15.1: :on: 
SLE 15.0: :on:

**Pull request merged**